charts/operator-wandb/templates/mysql.yaml

```diff
@@ -16,4 +16,17 @@ data:
   MYSQL_PASSWORD: {{ default (randAlphaNum 64) (include "wandb.mysql.password" .) | b64enc }}
 {{- end }}
 
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-mysql-configmap
+  labels:
+    {{- include "wandb.commonLabels" . | nindent 4 }}
+data:
+  MYSQL_PORT: "{{ include "wandb.mysql.port" . }}"
+  MYSQL_HOST: "{{ include "wandb.mysql.host" . }}"
+  MYSQL_DATABASE: "{{ include "wandb.mysql.database" . }}"
+  MYSQL_USER: "{{ include "wandb.mysql.user" . }}"
```

Comment:
Super duper nit and out-of-scope for this PR: We are inconsistent about ending files with newlines. Should we decide one way or the other and add a linter to enforce?

Reply:
We can think about this. I can add an issue and we can pick that up at a later point.

Reply:
https://github.com/wandb/helm-charts/issues/274 
Basic placeholder for now.